common/datavalues/src/types/type_timestamp.rs

```diff
@@ -169,12 +172,12 @@ impl DataType for DateTimeType {
     }
 }
 
-impl std::fmt::Debug for DateTimeType {
+impl std::fmt::Debug for TimeStampType {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         if self.precision() == 0 {
-            write!(f, "DateTime")
+            write!(f, "TimeStamp")
```

Comment:
So there will be always `(precision)` ?

Reply:
maybe fmt with always precision will be good ? 
and just add a alias `TIMESTAMP` => `TIMESTAMP(6)` will be ok?